allocate.go

```diff
@@ -424,6 +424,17 @@ func UserDataDir(dir string) ExecAllocatorOption {
 	return Flag("user-data-dir", dir)
 }
 
+// UserDataProfileDir is the command line option to set the profile
```

Comment:
- `UserDataProfileDir` --> `ProfileDirectory`
- `the profile subdirectory under the UserData directory` --> `the profile directory under  the user data directory`